pkg/network/ebpf/c/protocols/https.h

```diff
@@ -28,6 +30,25 @@ static __always_inline void https_process(conn_tuple_t *t, void *buffer, size_t
     read_into_buffer(http.request_fragment, buffer, len);
     http.owned_by_src_port = http.tup.sport;
     log_debug("https_process: htx=%llx sport=%d\n", &http, http.owned_by_src_port);
+
+    protocol_t *cur_fragment_protocol_ptr = bpf_map_lookup_elem(&dispatcher_connection_protocol, &http.tup);
+    if (cur_fragment_protocol_ptr == NULL) {
+        protocol_t cur_fragment_protocol = PROTOCOL_UNKNOWN;
+        conn_tuple_t inverse_conn_tup = http.tup;
+        flip_tuple(&inverse_conn_tup);
+
+        cur_fragment_protocol_ptr = bpf_map_lookup_elem(&dispatcher_connection_protocol, &inverse_conn_tup);
+
+        // If we've already identified the protocol of the socket, no need to read the buffer and try to classify it.
```

Comment:
rephrase the comment to match the code flow, now it "compliments" the code flow that is actually doesn't present in the code.

e.g: "try classifying the protocol if no prior identification exists"